qa/workunits/fs/full/subvolume_rm_retained_snapshot.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+set -ex
+
+# This testcase tests the scenario of the 'ceph fs subvolume rm --retained-snapshots'
+# mgr command when the osd is full.The osd is of the size 2GiB. The subvolume is
+# created and 1GB file is written. The full-ratios are set below 500MB such that the
+# osd is treated as full. Now the subvolume with retained-snapshots is removed. This
+# should fail with the introduction of FULL capabilities which the mgr holds as the
+# config update gets ENOSPACE error
+
+set -e
+expect_failure() {
+	if "$@"; then return 1; else return 0; fi
+}
+
+ceph fs subvolume create cephfs sub_0
+subvol_path=$(ceph fs subvolume getpath cephfs sub_0 2>/dev/null)
+
+#For debugging
+echo "Before write"
+df -h
+ceph osd df
```

Comment:
The echo message is more precise, it will be easier to understand it from the logs. See -

```suggestion
echo "Prininting system disk usages for host as well Ceph before writing on subvolume"
df -h
ceph osd df
```

The comment in above case won't be required anymore.